### PR TITLE
Updated Bangla Guide

### PR DESCRIPTION
**Pull request by shepherd-06**
Previous Bangla text was in proper format. It was probably translated directly using google translator.I have changed it to how people actually read/write in Bangla.

### CODE REVIEW

File: README-bn.md [hunk 1]
```diff
@@ -1,42 +1,43 @@
...
 - [ `简体 中文 的 介绍 文件`] (README-zhcn.md)
-- [ `Readme ই.এম. পর্তুগীজ (বিআর)`] (README-ptBR.md)
-- [ `Readme en Español (এলএ)`] (README-spLA.md)
+- [ `Readme em Português (BR)ে`] (README-ptBR.md)
+- [ `Readme en Español (LA)`] (README-spLA.md)
 - [ `Руководство на русском`] (README-ru.md)
 - [`Panduan dalam Bahasa Indonesia`](README-id.md)
 
-### কি করতে হবে তা নিশ্চিত না?
+### কি করতে চাচ্ছেন ওই বাপারে নিশ্চিত না?
```
Comment (SubhrajyotiSen) [inline README-bn.md]: Isn't `হবে` more appropriate than `চাচ্ছেন` in this case?

File: README-bn.md [hunk 2]
```diff
@@ -1,42 +1,43 @@
-## ওহে বিশ্ব
+## হেলো ওয়ার্ল্ড!
 
-আপনি একটি শিক্ষানবিস? আপনার পরীক্ষা খিঁচ অনুরোধ এখানে পাঠান! কেউ পর্যালোচনা করবে (মার্জ) তাদের.
+আপনি কি একজন শিক্ষানবিস? আপনি আপনার টেস্ট Pull Request এখানে পাঠাতে পারেন! কেউ একজন তা রিভিও করে মারজ করে দিবে।
```
Comment (SubhrajyotiSen) [inline README-bn.md]: I think মার্জ sounds like `merge` more than `মারজ`

**Comment by SubhrajyotiSen** [comment]
Thanks for correcting the translation 😄 
I will be reviewing later today and leave comment(if any)

**Review by SubhrajyotiSen — COMMENTED** [review]
Nice translation 👍 
Can you please check my comments once and provide your feedback?

File: README-bn.md (hunk 1 above)
Comment (ibtehaz-shawon) [reply README-bn.md]: Yes. It would be. 
Thank you.

File: README-bn.md (hunk 2 above)
Comment (shepherd-06) [reply README-bn.md]: Yes.. Sorry about that. That was a typo.

File: README-bn.md (hunk 1 above)
Comment (shepherd-06) [reply README-bn.md]: yeah... I will change that.

**Review by SubhrajyotiSen — APPROVED** [review]
This is a much better translation compared to the existing one. Thank you so much

**Pull request merged**